src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    switch (actions[i].type) {
```

Comment:
why do you need index in this loop? use `for of`
```suggestion
  for (const action of actions) {
    switch (action.type) {
```